hack/generated/Taskfile.yml

```diff
@@ -0,0 +1,113 @@
+version: '3'
+
+vars:
+  # careful here as these can conflict with the variables
+  # defined in generator/Taskfile.yml when included in ./Taskfile.yml
+  GENERATED_APP: k8sinfra-controller
+
+  ROOT_DIR:
+    sh: git rev-parse --show-toplevel
+
+  GENERATED_GO_DIRS:
+    sh: cd {{.ROOT_DIR}}/hack/generated && go list -f '{{"{{.Dir}}"}}' ./... 
+
+  GENERATED_GO_DIRS_TO_TEST:
+    # We exclude the apis folder because it's really large
+    # and test discovery takes a good amount of time (>10s)
+    sh: cd {{.ROOT_DIR}}/hack/generated && go list -f '{{"{{.Dir}}"}}' ./...  | grep -vE '/apis(/|$)'
+
+env:
+  KIND_CLUSTER_NAME: k8sinfra-generated
+
+tasks:
+  # Note: test-int can be run manually but is not included here so as to avoid slow
+  # tests bogging down the dev cycle
+  default:
+    desc: Perform all local pre-commit tasks.
+    deps: [generate-crds, header-check, format-code, test]
+  
+  # There is a ci specific target because we want the CI pass to fail if
+  # the code has not been go fmt-ed, whereas locally we want "make all"
+  # to just format the code for you
+  ci:
+    # no description as it is not for local use
+    # order must be explicit as CRDs must be generated before header check
+    cmds:
+    - task: generate-crds
+    - task: ci-rest
+
+  ci-rest:
+    # these can all be run in parallel
+    deps: [header-check, test-cover, test-integration-envtest]
+
+  lint:
+    desc: Run lint checks.
+    cmds: [golangci-lint run -v]
```

Comment:
Accidentally dropped the improvements I made to exclude the apis folder, or was that on purpose?

Reply:
restored